Detections/CommonSecurityLog/TimeSeriesAnomaly-MultiVendor_NetworkTraffic.txt

```diff
@@ -6,6 +6,16 @@
 //
 // Id: 06a9b845-6a95-4432-a78b-83919b28c375
 //
+// Severity: Medium
+//
+// QueryFrequency: 1h
+//
+// QueryPeriod: 30d
+//
+// AlertTriggerOperator: gt
+//
+// AlertTriggerThreshold: 0
```

Comment:
Maybe this should be 3 as that is what you are using in the series_decompose_anomalies function.

Reply:
Agreed per discussion to match scorethreshold with alerttriggerthreshold. 
Submitted changes to both files where scorethreshold is used.